torchci/pages/api/flaky-tests/disable.ts

```diff
@@ -77,8 +87,13 @@ export async function handleFlakyTest(test: FlakyTestData, issues: IssueData[],
 }
 
 
-export function getLatestWorkflowURL(workflow_ids: string[]): string {
-    return `https://github.com/pytorch/pytorch/actions/runs/${workflow_ids[workflow_ids.length - 1]}`;
+export function getLatestTrunkWorkflowURL(test: FlakyTestData): string {
+    let index = test.branches.lastIndexOf("master");
+    if (index < 0) {
+        console.warn(`Flaky test ${test.name} has no trunk failures. Disabling anyway, but this may be unintended.`);
```

Comment:
always thought trunk was kinda confusing for external people. Makes sense for internal but for external users master or main might be more suitable.

Reply:
Hmmm this "trunk" terminology was adopted during the master -> main migration as a common reference to the default branch. After we migrate pytorch/pytorch to main, I'll consider switching this to main.